client/client.go

```diff
@@ -305,10 +299,13 @@ func (c *Client) downloadTimestamp() error {
 func (c *Client) downloadSnapshot() error {
 	logrus.Debug("downloadSnapshot")
 	role := data.RoleName("snapshot")
+	if c.local.Timestamp == nil {
```

Comment:
Is this correct? Technically the inexistence of a time-stamp means that we can't proceed with the download of a snapshot, but the error seems to mean that something is wrong with snapshots itself, when in fact this error seems to be caused by the inexistence of `c.local.Timestamp`